docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -252,10 +257,12 @@ The simulation concept model consists of five main components:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
   - Location: United States
-  - Cohort type: Prospective closed cohort of individuals 15 years and older
+  - Cohort type: Prospective **open** cohort of individuals 15 years and older
```

Comment:
it is a **closed** cohort of individuals aged 15 years and older, but we will examine only the age 50+ population, which will effectively be an **open** cohort of individuals aged 50 and older.

Reply:
> it is a **closed** cohort of individuals aged 15 years and older, but we will examine only the age 50+ population, which will effectively be an **open** cohort of individuals aged 50 and older.

Yes, you are right!